Source/Core/AudioCommon/CMakeLists.txt

```diff
@@ -33,8 +33,17 @@ endif(PULSEAUDIO_FOUND)
 
 if(WIN32)
 	set(SRCS ${SRCS} XAudio2Stream.cpp)
+
+	add_dolphin_library(audiocommon_xaudio27 "XAudio2_7Stream.cpp" "${LIBS}")
```

Comment:
It needs different include directories. It's a bit special, if the normal headers are included, it won't build as it finds the up to date XAudio.h file and we want the older one.